backends/ebpf/tests/ptf/test.py

```diff
@@ -506,3 +506,52 @@ def runTest(self):
         pkt[Ether].type = 0xFF00
         testutils.send_packet(self, PORT0, pkt)
         testutils.verify_no_other_packets(self)
+
+
+class PSATernaryTest(P4EbpfTest):
+
+    p4_file_path = "p4testdata/psa-ternary.p4"
+
+    def runTest(self):
+        # flow rules for 'tbl_ternary_0'
+        # 1. ipv4.srcAddr=1.2.3.4/0xffffff00 => action 0 priority 1
+        # 2. ipv4.srcAddr=1.2.3.4/0xffff00ff => action 1 priority 10
+        self.table_add(table="ingress_tbl_ternary_0", keys=["1.2.3.4^0xffffff00"], action=0, priority=1)
```

Comment:
So ^ is the mask operator?
Would it make sense to use &&& to resemble P4 more?

Reply:
Yeah, the problem is that &&& collides with the bash "&". Since psabpf-ctl is primarily a CLI tool (the Python functions just wrap it), we found it better to use "^" as the mask operator.

Reply:
Is this documented someplace visible?